packages/terra-search-field/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+Changed
```

Comment:
This heading missing some syntax formatting

Reply:
Taking a closer look, this seems to have changed recently with the change log lint resolution PR. I think this may be correct as is, but I'd like to confirm with @mjhenkes on the changes. Whether this should be unformatted headings or triple #'s,

```
Changed
```

```
### Changed
````

Reply:
This is intended, repeating headings are not recommend because they will break hash linking to sections in the document.

Reply:
Sorry to go around on this but we ended up updating this to a list style, see #3106 for more details